### PR TITLE
change 'USER' to 2

### PR DESCRIPTION
**Pull request by ipwndwagons**
'user' doesnt work and it confuse me 😭

**Please describe the changes this PR makes and why it should be merged:**
cause 	.setType('USER'); doesnt work

### CODE REVIEW

File: guide/interactions/context-menus.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ const { ContextMenuCommandBuilder } = require('@discordjs/builders');
 
 const data = new ContextMenuCommandBuilder()
 	.setName('echo')
-	.setType('USER');
+	.setType(2);
```
Comment (thehairy) [inline guide/interactions/context-menus.md:18]:
```suggestion
	.setType(ApplicationCommandOptionTypes.USER);
```

File: guide/interactions/context-menus.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ const { ContextMenuCommandBuilder } = require('@discordjs/builders');
```
Comment (thehairy) [inline guide/interactions/context-menus.md:15]: idk if thats the right import tho
```suggestion
const { ApplicationCommandOptionTypes } = require('discord-api-types/v9')

```

**Review by thehairy — CHANGES_REQUESTED** [review]
Should use ENUM instead.

**Comment by megatank58** [comment]
this can also be left as it is until v14 rewrite, using enum here will be inconsistent with the rest of sections

**Comment by almostSouji** [comment]
> this can also be left as it is until v14 rewrite, using enum here will be inconsistent with the rest of sections

as far as i am aware this is currently not valid with the version of builders the stable 13.8.0 uses, so it cannot be left as-is, even if it means introducing some inconsistency

File: guide/interactions/context-menus.md (hunk 2 above)
Comment (megatank58) [reply guide/interactions/context-menus.md:15]: https://discord.js.org/#/docs/discord.js/stable/typedef/ApplicationCommandOptionType

Can't this be used instead here :thinking:

Comment (BenjammingKirby) [reply guide/interactions/context-menus.md:15]: builders exports a ContextMenuType enum right? just use that?

File: guide/interactions/context-menus.md (hunk 1 above)
Comment (almeidx) [reply guide/interactions/context-menus.md:18]: `ApplicationCommandType.User`

File: guide/interactions/context-menus.md (hunk 2 above)
Comment (thehairy) [reply guide/interactions/context-menus.md:15]: > https://discord.js.org/#/docs/discord.js/stable/typedef/ApplicationCommandOptionType
> 
> Can't this be used instead here 🤔

No, I have mistaken this for options, but its an ApplicationCommand type.

**Pull request merged**